pgmoon/init.moon

```diff
@@ -127,15 +128,26 @@ class Postgres
       @database = opts.database
       @port = opts.port
       @password = opts.password
+      @ssl = opts.ssl
+      @ssl_verify = opts.ssl_verify
+      @ssl_required = opts.ssl_required
+      @key = opts.key
+      @cert = opts.cert
+      @cafile = opts.cafile
 
   connect: =>
     @sock = socket.new!
     ok, err = @sock\connect @host, @port
     return nil, err unless ok
 
     if @sock\getreusedtimes! == 0
+      if @ssl
+        success, err = @send_ssl_message!
+        return nil, err unless success
```

Comment:
Yes we should return `true`! I added this just before submitting the patching without testing it, totally guilty.